authors/apps/articles/serializers.py

```diff
@@ -52,4 +52,42 @@ class Meta:
             'updated_at'
         )
 
+class CommentsSerializers(serializers.ModelSerializer):
+    body = serializers.CharField(
+       max_length = 200,
+       required = True,
+       error_messages = {
+           'required': 'Comments field cannot be blank'
+       }
+    )
+    def format_date(self, date):
+        return date.strftime('%d %b %Y %H:%M:%S')
+
+    def to_representation(self,instance):
+       """
+       overide representation for custom output
+       """
+       threads = [
+           {
 
+               'id': thread.id,
+                'body': thread.body,
+                'author': thread.author.username,
+                'created_at': self.format_date(thread.created_at),
+                'replies': thread.threads.count(),
+                'updated_at': self.format_date(thread.updated_at)}  for thread in instance.threads.all()]
```

Comment:
the closing curly bracker `}` should be in the next line, and the `]` should be on the next line after e.g
```
...
        'updated_at': self.format_date(thread.updated_at)
    } for thread in instance.threads.all()
]

```